build.zig

```diff
@@ -521,6 +538,41 @@ fn build_tigerbeetle_executable_multiversion(b: *std.Build, options: struct {
     return build_multiversion.addPrefixedOutputFileArg("--output=", basename);
 }
 
+// Downloads a pre-build llvm-objcopy from <https://github.com/tigerbeetle/dependencies>.
+fn build_tigerbeetle_executable_get_objcopy(b: *std.Build) std.Build.LazyPath {
+    const llvm_objcopy_artifact: struct { name: []const u8, checksum: u256 } =
+        switch (b.graph.host.result.os.tag) {
+        .linux => .{
+            .name = "llvm-objcopy-x86_64-linux",
+            .checksum = 0x3e2fe8f359c63eb62069e322f9bc079b2876301510afb15f70d117b30a2eea36,
```

Comment:
I did get a different checksum when building for Linux with your instructions [here](https://github.com/tigerbeetle/dependencies), but it seemed to be repeatable on my machine :thinking:.

Can you try build again, and see what checksum you get?

(mine was `925d140798f45f53878d47c8d2e9548893b71ac57edbe28f43c1ffa2aa0dc7d3  llvm-objcopy`)

Reply:
Huh, 

```
λ sha256sum llvm-objcopy
3e2fe8f359c63eb62069e322f9bc079b2876301510afb15f70d117b30a2eea36  llvm-objcopy
```

Let me try re-building this after removing my local containers, maybe docker re-uses old ubuntu:latest?

Reply:
Nope, still 3e2fe8f359c63eb62069e322f9bc079b2876301510afb15f70d117b30a2eea36

Reply:
Trying this command to pin the image now:

```
λ docker run -it --name llvm_build ubuntu:noble-20240801 /bin/bash -c "
      apt-get update && apt-get install -y git cmake ninja-build clang
      git clone --depth 1 --branch release/18.x https://github.com/llvm/llvm-project.git
      mkdir llvm-project/build && cd llvm-project/build
      cmake -G Ninja -DLLVM_ENABLE_PROJECTS=clang -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD=X86 -DLLVM_BUILD_STATIC=ON ../llvm
      ninja llvm-objcopy
  "

  docker cp llvm_build:/llvm-project/build/bin/llvm-objcopy .
  docker rm llvm_build
```

Reply:
Huh, I get that hash with `ubuntu:noble-20240801` and with `ubuntu:noble-20240530`!

Reply:
@cb22 tried this on a different laptop, and I still get 

```
λ sha256sum llvm-objcopy
3e2fe8f359c63eb62069e322f9bc079b2876301510afb15f70d117b30a2eea36  llvm-objcopy
```

Reply:
It looks very likely it was my `ubuntu:latest` that was very out of date! Could have sworn I saw it pulling, though! Will confirm output from the pinned version in a second.

Reply:
Woo

```
/tmp> sha256sum llvm-objcopy                                                      539.147s 15:59
3e2fe8f359c63eb62069e322f9bc079b2876301510afb15f70d117b30a2eea36  llvm-objcopy
```

Reply:
Pinned the version upstream:

https://github.com/tigerbeetle/dependencies/commit/983da8decafdb1616dc3695490dcc6d5939d9eb2